### PR TITLE
Fix flex values after blur changes

### PR DESCRIPTION
**Pull request by Palid**
Fixes https://github.com/vector-im/element-web/issues/18632

---
This change is marked as an *internal change* (Task), so will not be included in the changelog.

Preview: https://611f8d3761ab8339a13cf326--matrix-react-sdk.netlify.app
⚠️ Do you trust the author of this PR? Maybe this build will steal your keys or give you malware. Exercise caution. Use test accounts.

### CODE REVIEW

**Comment by Palid** [comment]
@nadonomy this should fix https://github.com/vector-im/element-web/issues/18632

File: res/css/structures/_MatrixChat.scss [hunk 1]
```diff
@@ -33,6 +33,7 @@ limitations under the License.
     height: 100%;
 }
 
+
```
Comment (dbkr) [inline res/css/structures/_MatrixChat.scss:36]: intentional?

**Review by dbkr — APPROVED** [review]
Thanks for the quick fix. Could https://github.com/vector-im/element-web/issues/18640 be related?

File: res/css/structures/_MatrixChat.scss (hunk 1 above)
Comment (Palid) [reply res/css/structures/_MatrixChat.scss:36]: accidental.

**Comment by Palid** [comment]
> Thanks for the quick fix. Could [vector-im/element-web#18640](https://github.com/vector-im/element-web/issues/18640) be related?

Kinda, but this PR doesn't fix it.

**Pull request merged**